libs/state/effects/src/lib/effects.service.ts

```diff
@@ -74,7 +74,7 @@ export class RxEffects implements OnDestroy, OnDestroy$ {
   private static nextId = 0;
   readonly _hooks$ = new Subject<DestroyProp>();
   private readonly observables$ = new Subject<Observable<unknown>>();
-  // we have to use publish here to make it hot (composition happens without subscriber)
+  // we have to use share here to make it hot (composition happens without subscriber)
```

Comment:
is this comment even true?

Reply:
no, it's not... https://stackblitz.com/edit/rxjs-nwerqp?file=index.ts

Reply:
probably it is meant in a different way

Reply:
Why it is not true? `share` will multicast the stream (hot), so the computation is only made once even if there are multiple subscribers. [Here is an example](https://stackblitz.com/edit/rxjs-lhe4qp?file=index.ts).

Reply:
`composition happens without subscriber` is not true for `share`, only for `connectable` after calling the `connect` method :)